tinygrad/helpers.py

```diff
@@ -223,8 +236,9 @@ def wrapper(*args, **kwargs) -> bytes:
 
 # *** http support ***
 
-def fetch(url:str) -> pathlib.Path:
+def fetch(url:str, ext:Optional[str]=None) -> pathlib.Path:
```

Comment:
Why do I need an extension?

Reply:
The `test/external/external_test_yolov8.py` was giving issues complaining that there's no ".pt" extension. I'll update the test to just rename the cache directly to have the extension.